src/index.ts

```diff
@@ -322,17 +326,31 @@ function main(): void {
 
   cacheAction.idslib.onMain(async () => {
     if (cacheAction.noopMode) {
-      actionsCore.warning(NOOP_TEXT);
+      actionsCore.warning(TEXT_NOOP);
       return;
     }
 
+    if (cacheAction.idslib.nixStoreTrust === "untrusted") {
+      actionsCore.warning(TEXT_TRUST_UNTRUSTED);
+      return;
+    } else if (cacheAction.idslib.nixStoreTrust === "unknown") {
+      actionsCore.info(TEXT_TRUST_UNKNOWN);
```

Comment:
Should this not be `warning`?

Reply:
I don't think so: it may be unknown if they're using a remote ssh store, for example. I don't want to emit useless warnings for scenarios we're not certain are worth a warning.